opensrp-fp/src/main/java/org/smartregister/fp/features/profile/view/ClientHistoryFragment.java

```diff
@@ -77,7 +77,7 @@ public void onItemClicked(HashMap<String, String> item, int position) {
         Utils.proceedToContact(baseEntityId, details, getActivity());
     }
 
-    static ClientHistoryFragment newInstance(Bundle bundle) {
+    public static ClientHistoryFragment newInstance(Bundle bundle) {
```

Comment:
Why have made this public?

Reply:
Codacy says: 

> Use explicit scoping instead of the default package private level

Reply:
I think using protected would have been better. But we can use that next time